trestle/core/commands/author/profile.py

```diff
@@ -319,6 +332,7 @@ def assemble_profile(
             version: Optional version for the assembled profile
             required_sections: Optional List of required sections in assembled profile, as comma-separated short names
             allowed_sections: Optional list of section short names that are allowed, as comma-separated short names
+            default_namespace: Optional namespace to be used for properties that don't have namespace specfied
```

Comment:
I don't see how this namespace is used in the assemble ?

Reply:
it isn't yet but it will be finished after demo